stargz/stargz.go

```diff
@@ -390,6 +406,9 @@ func (r *Reader) Lookup(path string) (e *TOCEntry, ok bool) {
 	// here? And it probably needs a link count field stored in
 	// the TOCEntry.
```

Comment:
Is this TODO now done?

Reply:
Yes, this is done. So I deleted the comment.